lib/std/os/uefi/protocol.zig

```diff
@@ -18,7 +18,7 @@ pub const GraphicsOutput = @import("protocol/graphics_output.zig").GraphicsOutpu
 
 pub const edid = @import("protocol/edid.zig");
 
-pub const SimpleNetwork = @import("protocol/simple_network.zig").SimpleNetwork;
+pub const SimpleNetworkMode = @import("protocol/simple_network.zig").SimpleNetwork;
```

Comment:
Why rename this?

Reply:
thats what all the uses of that type called it, very naive fix like i said. but yeah in hindsight the rename probably shouldve been the other way around.